slack-base/src/main/java/com/hubspot/slack/client/models/events/user/SlackMemberLeftChannelEventIF.java

```diff
@@ -0,0 +1,34 @@
+package com.hubspot.slack.client.models.events.user;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.ChannelType;
+import com.hubspot.slack.client.models.events.SlackEvent;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+@JsonDeserialize(as = SlackMemberLeftChannelEvent.class)
```

Comment:
Why do we need this?

Reply:
I mean, I see that it's copied from other events but still, why do we need this?

Reply:
Okay, I see. Parent interface has custom deserializer.